clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/manifest/KubernetesDeployManifestOperation.java

```diff
@@ -251,15 +249,6 @@ private void applyTraffic(KubernetesManifestTraffic traffic, KubernetesManifest
     traffic.getLoadBalancers().forEach(l -> attachLoadBalancer(l, target));
   }
 
-  private void validateManifestsForRolloutStrategies(List<KubernetesManifest> manifests) {
-    long numReplicaSets =
-        manifests.stream().filter(m -> m.getKind().equals(KubernetesKind.REPLICA_SET)).count();
-    if (description.getStrategy() != null && numReplicaSets != 1) {
-      throw new RuntimeException(
-          "Spinnaker can manage traffic for one ReplicaSet only. Please deploy one ReplicaSet manifest or disable rollout strategies.");
```

Comment:
I considered leaving the error if _zero_ ReplicaSet manifests are being deployed with a non-null strategy config, but then decided not to since it won't actually break anything downstream. I could be swayed either way, though. :woman_shrugging:

Reply:
Removing the error entirely seems reasonable to me!